app/models/katello/content_view.rb

```diff
@@ -299,7 +299,7 @@ def repositories_to_publish_by_library_instance
     end
 
     def duplicate_repositories_to_publish
-      repositories_to_publish_by_library_instance.select { |_key, val| val.count > 1 }.keys
+      repositories_to_publish_by_library_instance.select { |key, val| val.count > 1 && key.present? }.keys
```

Comment:
this seems to work in my tests, but I 'd like a clearer ```return [] unless composite?``` rather than relying on library instance id  and hence the key being nil for a non composite.